marc_to_solr/lib/princeton_marc.rb

```diff
@@ -827,3 +827,10 @@ def siku_heading?(field)
 def any_thesaurus_match?(field, thesauri)
   field.any? { |subfield| subfield.code == '2' && thesauri.include?(subfield.value) }
 end
+
+def valid_linked_fields(record, field_tag, accumulator)
+  fields = record.fields(field_tag).select { |f| f["w"] =~ /99[0-9]+6421/ }
```

Comment:
Can you put a comment explaining the regex? That it's differentiating between Alma-styled MMS ID, NJP prefix, and OCLC ids? (that's what I understood from checkin)

Reply:
In essence, the regex ensures that an MMS ID appears somewhere in the $w, and when a prefix is added to the MMS ID (like NjP or PUL), it'll still work. I agree it would be a good idea to have a small comment in the code.